### DISCUSSION TITLE
"disableVerticalSwipes" does not work in the latest update 21.4

### DISCUSSION THREAD

**Discussion by Daytly** [python-telegram-bot/python-telegram-bot#4377] [Feature Suggestions]
"disableVerticalSwipes" does not work in the latest update 21.4. For some reason, the version in the web application is 7.6

**Comment by Poolitzer** [comment]
Hi, this is depending on the app you open the web app with, not our library.

**Reply by Daytly** [reply]
Sorry, I didn't know. Thank you very much.
